docs/guides/communication_setting/communication_setting_create.md

```diff
@@ -144,71 +144,73 @@ resource "conformity_communication_setting" "comm_setting" {
     url = ""
   }
 
+  service_now {
```

Comment:
If this is just repeating what is in example/communication_setting/template.tf, I think we simply link to that file, rather than duplicating